docs/LwM2M_Client/Nordic/nRF7002.md

```diff
@@ -97,13 +98,10 @@ To connect the board:
         <br/>
         ![Anjay set configuration](images/anjay_config_set.PNG "Anjay set configuration"){:style="float: left;margin-right: 1177px;margin-top: 7px; margin-bottom: 17px;"}
 
+
     !!! note
-        **nRF7002** connects to the Internet via a WiFi network. Remember to set the credentials of your network using `anjay config set wifi_ssid <value>` and `anjay config set wifi_password <value>`.
+        **nRF7002 DK** connects to the Internet via a Wi-Fi network. Remember to set the credentials of your network using `anjay config set wifi_ssid <value>` and `anjay config set wifi_password <value>`.
+        ![Set WiFi credentials](images/set_wifi_credentials.png "Anjay set configuration"){:style="float: left;margin-right: 1177px;margin-top: 7px; margin-bottom: 17px;"}
```

Comment:
I don't see an image. Wrong folder or something.
Also, I can't find the image anywhere so did you push it?

Reply:
That is true, I haven't